src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
```

Comment:
No need to state where the class is used. (e.g. 'in the address book.` is unnecessary)